### ISSUE TITLE
Installation with CPU only

### ISSUE THREAD

**Issue by miquelduranfrigola** [CSUBioGroup/PGMG#9]
Hi,

Thanks for a great tool. I would like to use PGMG on CPU-only (Ubuntu) but, unfortunately, I do not find installation instructions.
Are those available?

Thanks in advance!

**Comment by filipsPL** [comment]
Any success so far with that?

**Comment by miquelduranfrigola** [comment]
Not really... I will try again in a few weeks when I retake the project. Any advice that the team can give? Thanks!

**Comment by e-yi** [comment]
Hi, sorry for the late reply. We did not test our code in the CPU-only environment, but I think it should work just fine. For installation, using the cpu-only version of torch and dgl will be enough. Did you encounter any particular problem?
